### PR TITLE
Use markdown links

### PR DESCRIPTION
**Pull request by alphapapa**
See <https://github.com/scientific-python/scientific-python-hugo-theme/issues/568>.

### CODE REVIEW

File: content/about/governance.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ to allow existing members time to voice objections.
 Community Leaders are Community Managers who have additional responsibilities to ensure the smooth running of the Scientific Python project.
 Community Leaders are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to the Scientific Python project.
 (Funding to community members/projects is theirs to pursue and manage.)
-Community Leaders should ensure smooth progress on the big-picture [roadmap]({{< relref "roadmap.md" >}}).
+Community Leaders should ensure smooth progress on the big-picture [roadmap](roadmap).
```
Comment (jarrodmillman) [inline content/about/governance.md:45]:
```suggestion
Community Leaders should ensure smooth progress on the big-picture [roadmap](/roadmap).
```

File: content/code_of_conduct.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Scientific Python Ecosystem Core Team directly, or ask the Team for advice, in
 confidence.
 
 You can report issues to the
-[Scientific Python Community Managers]({{% relref "/about#community-managers" %}}),
+[Scientific Python Community Managers](/about#community-managers),
```
Comment (jarrodmillman) [inline content/code_of_conduct.md:113]:
```suggestion
[Scientific Python Community Managers](/about/#community-managers),
```

File: content/grants/planning_next_decade/_index.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ for the next decade of data science, we will:
 
 Our initial focus is on creating a recommendation infrastructure for
 community-wide policy and on improving common engineering infrastructure
-(see the [SPEC]({{< relref "specs" >}}) documents).
+(see the [SPEC](specs) documents).
```
Comment (jarrodmillman) [inline content/grants/planning_next_decade/_index.md:33]:
```suggestion
(see the [SPEC](/specs) documents).
```

**Review by jarrodmillman — CHANGES_REQUESTED** [review]

**Review by jarrodmillman — APPROVED** [review]

File: content/grants/planning_next_decade/_index.md (hunk 3 above)
Comment (alphapapa) [reply content/grants/planning_next_decade/_index.md]: @jarrodmillman For my own edification, is that leading slash necessary or just a convention we're using?

Comment (jarrodmillman) [reply content/grants/planning_next_decade/_index.md]: Necessary. It represents the baseURL. Without it you get this error:
```
3:06:16 AM:   ✖ FAIL load public/grants/planning_next_decade/specs
3:06:16 AM:   | operator: load
3:06:16 AM:   | expected: 200 public/grants/planning_next_decade/specs
3:06:16 AM:   |   actual: ENOENT: no such file or directory, open '/opt/build/repo/public/grants/planning_next_decade/specs'
3:06:16 AM:   |       at: public/grants/planning_next_decade/index.html:148:19 <a href="specs">...</a>
```

Comment (jarrodmillman) [reply content/grants/planning_next_decade/_index.md]: See the error for 4f9bb25.

**Pull request merged**